tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -90,6 +102,21 @@ execute: |
 
   # Retrieve kernel
   snap download --basename=pc-kernel --channel="$version/edge" pc-kernel
+
+  # THIS IS A HACK
+  # TODO: Remove when pc-kernel snapd-info includes snap-bootstrap from snapd 2.68+
+  # Inject snapd-info file to account for kernels that don't yet include snapd 2.68+
+  unsquashfs -d tmp-pc-kernel pc-kernel.snap
+  cat <<EOF > tmp-pc-kernel/snapd-info
```

Comment:
we're doing this unconditionally atm, but perhaps the test should check for snapd-info presence and that VERSION >= 2.68 (prob ok to `echo $VERSION | cut -d. -f2` and only compare the minor number.